src/main/java/net/conjur/api/Credentials.java

```diff
@@ -13,17 +15,27 @@
 public class Credentials {
     private static final String CONJUR_AUTHN_LOGIN_PROPERTY = "CONJUR_AUTHN_LOGIN";
     private static final String CONJUR_AUTHN_API_KEY_PROPERTY = "CONJUR_AUTHN_API_KEY";
+    private static final String CONJUR_AUTHN_SERVICE_ID_PROPERTY = "CONJUR_AUTHN_SERVICE_ID";
```

Comment:
The current precedent for this is to set a `CONJUR_AUTHN_URL` property rather than specifying the service id:
https://github.com/cyberark/conjur-api-ruby/blob/master/lib/conjur/configuration.rb#L298

I'm not saying this is a wrong approach, but I think it warrants more consideration for what should be the expected interface for configuring Conjur client SDKs, before having two SDKs go with two different methods.

Reply:
`CONJUR_AUTHN_URL` it is!